el8/Dockerfile.buildtime

```diff
@@ -0,0 +1,18 @@
+FROM @BASE_IMAGE_NAME@
+LABEL maintainer="CMS Build"
+LABEL name="CMS Worker Node on EL - Buildtime"
+
+RUN dnf install --disablerepo=epel -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exporter perl-File-Path \
+          perl-File-Temp perl-Getopt-Long perl-IO perl-Text-ParseWords perl-constant perl-Digest-MD5 \
+          perl-PathTools perl-Data-Dumper perl-Digest-MD5 perl-Exporter perl-File-Path perl-File-Temp \
+          perl-Getopt-Long libdrm perl-File-Which perl-Scalar-List-Utils perl-Term-ANSIColor perl-CGI \
+          perl-DBI perl-Encode perl-Memoize perl-Storable perl-Time-HiRes perl-Time-Local perl-YAML \
+          perl-IO-Compress &&\
+    dnf install --disablerepo=epel -y tar rpm-build git nspr nss rsync wget openssl-devel ncurses-devel libX11-devel \
+          readline-devel tcl-devel tk-devel mesa-libGLU-devel libXext-devel libXft-devel \
+          libXpm-devel java-1.8.0-openjdk-devel java-11-openjdk-devel libtool gcc-c++ &&\
+    epel_pkgs="@EPEL_PACKAGES@" &&\
+    if [ -n "$epel_pkgs" ] ; then dnf install -y $epel_pkgs; fi &&\
+    xcmd="@EXTRA_COMMAND@" &&\
+    if [ "$xcmd" = "" ] ; then xcmd=true; fi &&\
+    eval $xcmd
```

Comment:
I think you also need the following line
- To have the build time stamp: https://github.com/cms-sw/cms-docker/blob/master/el8/Dockerfile.runtime#L31
- To have the container build date label : https://github.com/cms-sw/cms-docker/blob/master/el8/Dockerfile.runtime#L34

Reply:
you also need to add https://github.com/cms-sw/cms-docker/blob/master/el8/Dockerfile.runtime#L23-L25 so that when ever this image is rebuild then it updates the certificates